python/cac_tripplanner/templates/home.html

```diff
@@ -147,15 +147,26 @@ <h5 class="destination-address-2">{{ destinations.3.city }}, {{ destinations.3.s
         <div class="container">
             <div class="row">
                 <div class="col-sm-8">
-                    <h3>GoPhillyGo Spotlight <small><a href="#">View all &raquo;</a></small></h3>
-                    <div class="row">
+                    <h3>GoPhillyGo Spotlight <small>
+                        <a id="viewAllArticles" href="#">View all &raquo;</a>
+                    </small></h3>
+                    <div id="articlesSpinner" class="sk-spinner sk-spinner-three-bounce hidden">
+                        <div class="sk-bounce1"></div>
+                        <div class="sk-bounce2"></div>
+                        <div class="sk-bounce3"></div>
+                    </div>
+                    <div class="row articles">
                       {% if community_profile %}
                         <div class="col-sm-6">
                           <a class="block block-spotlight"
                              href="{% url 'cms.views.community_profile_detail' community_profile.slug %}">
                                 <h3 class="destination-name">{{ community_profile.title }}</h3>
                                 <h5>Community Profile</h5>
+                                {% if community_profile.narrow_image %}
+                                <img src="{{ community_profile.narrow_image }}" width="400" height="600"/>
+                                {% else %}
                                 <img src="https://placehold.it/400x600.jpg" />
```

Comment:
Cool. Might make sense then to add an issue to cleanup all of the if/elses that are sprinkled through the codebase.